composer.json

```diff
@@ -13,7 +13,7 @@
         "doctrine/inflector":   "~1.1",
         "doctrine/orm":         "~2.4",
         "symfony/filesystem":   "^3.0.2|~2.5|^4.0",
-        "twig/twig":            "^2.0||^1.28"
+        "twig/twig":            "~2.7"
```

Comment:
Why are we removing support for 1.x?

The security issue was fixed in both 1.38 and 2.7.

`^2.7.0||^1.38.0` should suffice.

Reply:
I was under the impression ^1.38 would not be namespaced. It is, so I will add it.

Will also use the `^`-notation, as it seems like the BC was unintentional.